tests/auto/rt_auto.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash --login
+set -eux
+if [ -f "accesstoken.sh" ]; then
+  source ./accesstoken.sh
+else
+  echo "Please create accesstoken.sh (600) with the following content\n"
+  echo "export ghapitoken=<GitHub API Token Here>"
+  exit 1
+fi
+
+export RT_COMPILER='intel'
+source ../detect_machine.sh
+echo "Machine ID: "+$MACHINE_ID
+if [[ $MACHINE_ID = hera.* ]]; then
+  WORKDIR=/scratch1/NCEPDEV/nems/Brian.Curtis/test
+  export PATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/bin:$PATH
+  export PYTHONPATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
+elif [[ $MACHINE_ID = orion.* ]]; then
+  WORKDIR=/work/noaa/nems/bcurtis/test
+  export PATH=/work/noaa/nems/emc.nemspara/soft/miniconda3/bin:$PATH
+  export PYTHONPATH=/work/noaa/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
+elif [[ $MACHINE_ID = jet.* ]]; then
+  WORKDIR=/lfs4/HFIP/h-nems/Brian.Curtis/test
+  export ACCNR="h-nems"
+  export PATH=/lfs4/HFIP/hfv3gfs/software/ecFlow-5.3.1/bin:$PATH
+  export PYTHONPATH=/lfs4/HFIP/hfv3gfs/software/ecFlow-5.3.1/lib/python2.7/site-packages
```

Comment:
This has recently been updated to use the miniconda version, but we can make this change in a future PR. Or you do it now, see current `rt.sh`:
```
  export PATH=/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/envs/ufs-weather-model/bin:/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/bin:$PATH
  export PYTHONPATH=/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/envs/ufs-weather-model/lib/python3.8/site-packages:/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/lib/python3.8/site-packages
  ECFLOW_START=/lfs4/HFIP/hfv3gfs/software/miniconda3/4.8.3/envs/ufs-weather-model/bin/ecflow_start.sh
```

Reply:
I will add it now.